ex7/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python
+
+copy ./flask /app
```

Comment:
* upper case for docker file commands
* this command is meaningless if you install the flask using pip install- you don't even access this folder /app/flask